doc/plotting/index.rst

```diff
@@ -501,17 +501,20 @@ Or you can save it to an html file::
 Interactive visualization of statistical map slices
 ---------------------------------------------------
 
-:func:`view_stat_map`: open stat map in a Papaya viewer (https://github.com/rii-mango/Papaya)::
+:func:`view_stat_map`: open stat map in a Brainsprite viewer (https://github.com/simexp/brainsprite.js)::
 
     >>> from nilearn import plotting, datasets     # doctest: +SKIP
-    >>> img = datasets.fetch_localizer_button_task()['tmaps'][0]     # doctest: +SKIP
-    >>> view = plotting.view_stat_map(img, threshold=2, vmax=4)     # doctest: +SKIP
+    >>> img = datasets.fetch_localizer_button_task()['tmap']     # doctest: +SKIP
+    >>> html_view = plotting.view_stat_map(img, threshold=2, vmax=4, cut_coords=[-42, -16, 52], title="Motor contrast")     # doctest: +SKIP
```

Comment:
This line is too long, and as a result it breaks HTML layout.
https://3522-1235740-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/plotting/index.html#interactive-visualization-of-statistical-map-slices

Line continuation in doctests is "..."

Reply:
Yeah i observed this just now. Will add this to the list.